Untitled Pull Request

Ethan-mo opened the pull request:
✨[feat]
1. 임시 Model을 구현하여, 게시글이 있는 모습을 구현하였다.
    1) CollectionView를 사용하였으며, MVVM패턴이 익숙하지않아, 임시적으로 Cell.swift를 생성하여 구현하였습니다.

jkim68888 committed:
[Style] 로그인화면 뷰세팅 완료

- UIButton Extension 추가
- 다이어리 폰트 추가
- 앱진입점 변경
- 로그인화면 뷰 완료

jkim68888 committed:
[fix] 버튼 이미지 옵셔널로 변경

jkim68888 committed:
[fix] 버튼 width 구하는 코드 수정

jkim68888 committed:
[feat] 로그인 임시 구현

- 로그인여부에 따른 루트뷰 변경

Ethan-mo committed:
Merge remote-tracking branch 'refs/remotes/origin/main'

Ethan-mo committed:
✨[feat]
1. 임시 Model을 구현하여, 게시글이 있는 모습을 구현하였다.
	1) CollectionView를 사용하였으며, MVVM패턴이 익숙하지않아, 임시적으로 Cell.swift를 생성하여 구현하였다.

Ethan-mo committed:
🎨[style]
1. 약간의 UI Frame변경

jkim68888 commented:
- 홈뷰 -> 현재 콜렉션뷰만 스크롤되는데, 전체뷰가 스크롤되게끔 해주세요
    (헤더랑 컬렉션뷰를 테이블뷰에 넣기 또는 컬렉션뷰 - 컴포지셔널 레이아웃 방법 등)

- 홈뷰 네비게이션바 -> 히든 시켜주세요

Ethan-mo committed:
🎨[style] + ✨[feat]
1. 지정한 배경 색상으로 통일
2. 지정한 폰트사용
3. Cell에서 PostViwer로 데이터 전달
